server/app/models/TrustedIntermediaryGroup.java

```diff
@@ -53,7 +55,8 @@ public String getDescription() {
     return this.description;
   }
 
-  public ImmutableList<Account> getManagedAccounts(Optional<String> search) {
+  public ImmutableList<Account> getManagedAccounts(
+      Optional<String> search, Optional<String> searchDate) {
```

Comment:
suggestion (blocking): change `Optional<String> searchDate` to `Optional<LocalDate> searchDate`. Do not do parsing and validation in a model class